src/NuGet.Clients/NuGet.VisualStudio.Client/source.extension.vsixmanifest

```diff
@@ -1,23 +1,53 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="NuGet.72c5d240-f742-48d4-a0f1-7016671e405b" Version="16.99.99" Language="en-US" Publisher="Microsoft Corporation" />
-        <DisplayName>NuGet Package Manager for Visual Studio 2019</DisplayName>
+        <Identity Id="NuGet.72c5d240-f742-48d4-a0f1-7016671e405b" Version="17.99.99" Language="en-US" Publisher="Microsoft Corporation" />
+        <DisplayName>NuGet Package Manager for Visual Studio 2022</DisplayName>
         <Description>A collection of tools to automate the process of downloading, installing, upgrading, configuring, and removing packages from a VS Project.</Description>
         <PackageId>Microsoft.VisualStudio.NuGet.Core</PackageId>
         <License>eula.rtf</License>
         <AllowClientRole>true</AllowClientRole>
     </Metadata>
     <Prerequisites>
-        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[16.0,17.0)" DisplayName="Visual Studio core editor" />
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[17.0,18.0)" DisplayName="Visual Studio core editor" />
     </Prerequisites>
     <Installation SystemComponent="true" InstalledByMsi="false" AllUsers="true">
-        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[16.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.VWDExpress" Version="[16.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.VPDExpress" Version="[16.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.VSWinExpress" Version="[16.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.VSWinDesktopExpress" Version="[16.0,17.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[17.0,18.0)">
+            <ProductArchitecture>amd64</ProductArchitecture>
+        </InstallationTarget>
+        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[17.0,18.0)">
+            <ProductArchitecture>x86</ProductArchitecture>
+        </InstallationTarget>
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0,18.0)">
+            <ProductArchitecture>amd64</ProductArchitecture>
+        </InstallationTarget>
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0,18.0)">
+            <ProductArchitecture>x86</ProductArchitecture>
```

Comment:
From the `Notes` section in https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/21382/How-to-upgrade-VSIX-to-Dev17?anchor=notes, I see that `If your extension needs to support Dev17 32-bit, simply extend your dev16 installation target to 17.0 and add x86 as the product architecture. `
``` xml
<InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0, 18.0)" >
  <ProductArchitecture>x86</ProductArchitecture>
</InstallationTarget>
```
I see that `version` starts from `17.0` here and I expect it to be `Version="[16.0, 18.0)" for x86 architecture`. Am I missing something?

Reply:
Although the current POR is that Dev17 will be 64-bit only, if enough customers give feedback that they need 32-bit we could decide to ship both 64-bit and 32-bit. So 32-bit VS is still getting basic (P0) testing division-wide.

The [16.0, 18.0) seems like a typo to me. Due to the interop changes in Dev17, release-6.0.x is not compatible with anything before 17.0. Is there any reason that we would want to install recent dev builds into an old version of VS?

Reply:
> Is there any reason that we would want to install recent dev builds into an old version of VS?

There is no reason AFAIK. `<ProductArchitecture>x86</ProductArchitecture>` setting made me to ask this question.